boltdesignsystem/bolt
# [DO NOT MERGE] DS-744 refine theme colors for table

## Jira

https://pegadigitalit.atlassian.net/browse/DS-744

## Summary

Colors were updated for the table body and headers.

## Details

- This was merged with table ticket [(Table component rework)](https://github.com/boltdesignsystem/bolt/pull/2437)
- Updated CSS color shadows
- Updated CSS table with new table headers colors

## How to test

- Pull the branch
- Review the code
- Check if table colors have good contrast
- Compare previous [table colors](https://feature-ds-516-table-component-rework.boltdesignsystem.com/pattern-lab/?p=visual-styles-theming-system) with the local branch and see which visuall and accessibility contrast aspects are better.
- check the Test page in the test folder (`/pattern-lab/?p=viewall-tests-table`) where are tables created for every theme color and their headers are sticky.
